docs/compatibility.md

```diff
@@ -711,9 +711,8 @@ to `false`.
 
 ### Float to String
 
-The GPU will use different precision than Java's toString method when converting floating-point data
-types to strings. The GPU uses a lowercase `e` prefix for an exponent while Spark uses uppercase
-`E`. As a result the computed string can differ from the default behavior in Spark.
+The GPU use [ryu](https://github.com/ulfjack/ryu) as the solution when converting floating-point data
```

Comment:
nit: The Rapids Accelerator for Apache Spark uses a method based on [ryu](https://github.com/ulfjack/ryu) when converting floating point data type to string. ...

Reply:
done.